SDPi_Supplement/asciidoc/volume1/tf1-ch-10-sdpi-p.adoc

```diff
@@ -12,47 +11,47 @@ The establishment of a trusted ecosystem of medical and non-medical devices and
+a| *SDPi Supplement Version Note*: For SDPi 1.0 some Actors and transactions have been deferred to a subsequent version, but are included here for completeness.
+Specifically:  SOMDS FHIR Gateway, SOMDS Sensor Gateway & SOMDS Smart App Platform, have been deferred.
 
 Deferred transactions have been so indicated in the transactions table.
 
 |===
 
-This section defines the actors, transactions, and/or content modules in this profile.
-General definitions of actors are given in the https://profiles.ihe.net/GeneralIntro/ch-A.html[_Technical Frameworks General Introduction Appendix A_].
+This section defines the Actors, transactions, and/or content modules in this profile.
+General definitions of Actors are given in the https://profiles.ihe.net/GeneralIntro/ch-A.html[_Technical Frameworks General Introduction Appendix A_].
 IHE Transactions can be found in the https://profiles.ihe.net/GeneralIntro/ch-B.html[_Technical Frameworks General Introduction Appendix B_].
 Both appendices are located at https://profiles.ihe.net/GeneralIntro/[_profiles.ihe.net/GeneralIntro_].
 
-<<vol1_figure_sdpi_p_actor_diagram>> shows the actors directly involved in the SDPi-P Profile.
+<<vol1_figure_sdpi_p_actor_diagram>> shows the Actors directly involved in the SDPi-P Profile.
 The relevant transactions between them are detailed in the subsequent <<vol1_table_sdpi_p_actors_transactions>>.
 Abstract Actors (i.e., those that provide common specifications that are utilized in other “concrete” or implementation actors) are indicated by stereotype names in italics (e.g., "<< _SOMDS_Participant_ >>".   The actors that inherit their capabilities include the stereotype at the top of their actor box.
-Alternatively, in accordance with traditional IHE style, the abstract actor's name can be in italics with "{_abstract_}" (e.g., see _SOMDS Connector_ in <<vol1_figure_sdpi_p_actor_diagram>>).
+Alternatively, in accordance with traditional IHE style, the Abstract Actor's name can be in italics with "{_abstract_}" (e.g., see _SOMDS Connector_ in <<vol1_figure_sdpi_p_actor_diagram>>).
 Actor groupings, including abstract with concrete are detailed in <<vol1_clause_sdpi_p_required_actor_groupings>>.
 
 [#vol1_figure_sdpi_p_actor_diagram]
 .SDPi-P Actor Diagram
 
 image::../images/vol1-diagram-sdpi-p-actor.svg[]
 
-<<vol1_table_sdpi_p_actors_transactions>> lists the transactions for each actor directly involved in the SDPi-P Profile.
-To claim compliance with this profile, an actor shall support all required transactions (labeled “R”) and may support the optional transactions (labeled “O”).
-Note that “Consumer” is indicated for actors that receive but do not directly respond to a specific transaction.
+<<vol1_table_sdpi_p_actors_transactions>> lists the transactions for each Actor directly involved in the SDPi-P Profile.
+To claim compliance with this profile, an Actor shall support all required transactions (labeled “R”) and may support the optional transactions (labeled “O”).
```

Comment:
```suggestion
To claim conformity with this profile, an Actor shall support all required transactions (labeled “R”) and may support the optional transactions (labeled “O”).
```

AFAIK compliance can be claimed when adhering to laws, in terms of standards manufacturers demonstrate conformity, right?